src/default-reducers.js

```diff
@@ -94,53 +127,53 @@ export function retrieveManyAborted(state, action) {
   };
 }
 
-export function retrieveManyResetResolution(state, action) {
+export function retrieveManyResetResolution(idAttr, state, action) {
   return {
     ...state,
     resourcesListMeta: {
-      retrievingStatus: null
+      retrievingStatus: resourceStatuses.NULL
```

Comment:
Why `NULL` for a reset? Also since I've not used reset, just wondering what the use case for it would be